packages/commonwealth/package.json

```diff
@@ -62,7 +62,8 @@
     "cosmos:start": "chmod u+x test/util/cosmos-chain-testing/v1/start.sh && ./test/util/cosmos-chain-testing/v1/start.sh && chmod u+x test/util/cosmos-chain-testing/v1beta1/start.sh && ./test/util/cosmos-chain-testing/v1beta1/start.sh && chmod u+x test/util/cosmos-chain-testing/ethermint/start.sh && ./test/util/cosmos-chain-testing/ethermint/start.sh",
     "cosmos:stop": "chmod u+x test/util/cosmos-chain-testing/v1/stop.sh && ./test/util/cosmos-chain-testing/v1/stop.sh && chmod u+x test/util/cosmos-chain-testing/v1beta1/stop.sh && ./test/util/cosmos-chain-testing/v1beta1/stop.sh && chmod u+x test/util/cosmos-chain-testing/ethermint/stop.sh && ./test/util/cosmos-chain-testing/ethermint/stop.sh",
     "publish-rmq-msg": "ts-node -r tsconfig-paths/register ./scripts/publishRmqMessageScript.ts",
-    "build-contracts": "npx hardhat compile"
+    "build-contracts": "npx hardhat compile",
+    "start-prod": "NODE_ENV=production node --enable-source-maps ./build/server.js"
```

Comment:
Two things here:

1. If we are just using this for local development, I think this will need NO_SSL=true, otherwise it will fail on most dev's machines, we haven't set up ssl certificates on our local DB.
2. This replaces start-external-webpack. Could you please remove it along with the associated entry in the startup-scripts.md?

Reply:
This is a production mode script (node from /build). Removing for now.

I have another ticket coming up in the bucket that will deal with "dev" and "prod" scripts, we need to define the conventions and update the docs.